README.md

```diff
@@ -40,10 +40,10 @@ Documentation for the components of the toolkit is provided in [`./docs`](./docs
 As a brief overview, the toolkit takes midi files as input and first converts
 them to a standard data structure like this:
 ```
-onset,track,pitch,dur
-0    ,0    ,100  ,250
-250  ,0    ,105  ,255
-250  ,1    ,100  ,100
+onset, track, pitch, dur, velocity
```

Comment:
Why don't we sub out the code block for a markdown table:

i.e. remove the triple backticks and replace this section with:

onset|track|pitch|dur|velocity
---|---|---|---|---
0|0|100|250|80
250|0|105|255|100
250|1|100|100|95

i.e.
```
onset|track|pitch|dur|velocity
---|---|---|---|---
0|0|100|250|80
250|0|105|255|100
250|1|100|100|95
```

Reply:
Didn't know this was possible -- I'll change it.